### ISSUE TITLE
ATS 1.48 UPDATE

### ISSUE THREAD

**Issue by ROSZCO** [dariowouters/ts-fmod-plugin#13]
Just letting you know ATS has been updated to 1.48, The mod no longer works it crashers the game due to incompatibility.
Thank so much.

**Comment by rackgaming** [comment]
Yeah, very unfortunate, I really hope he updates the FMOD, I use it to get on TruckersMP or convoys to use my personal preference sounds.
